node/src/chain_spec.rs

```diff
@@ -142,14 +357,19 @@ fn testnet_genesis(
 		},
 		balances: BalancesConfig {
 			// Configure endowed accounts with initial balance of ENDOWED_FUNDING and allocate the treasury TREASURY_FUNDING_PERCENT of total supply .
-			balances: endowed_accounts.iter().cloned().map(|k| {
-			 if k == treasury_account_id()
-			 {
-				 (k, treasury_funding)
-			 } else {
-				 (k, ENDOWED_FUNDING)
-			 }
-			}).collect(),
+			balances: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| {
+					if k == treasury_account_id() {
+						(k, treasury_funding)
+					} else if k == CrannyKeys::root() {
+						(k, 10_000_000__000_000_000_000)
+					} else {
+						(k, ENDOWED_FUNDING)
+					}
+				})
+				.collect(),
```

Comment:
Congrats to your first homegrown rust-snippet! 🥳

Reply:
Thanks :)